contracts/interchain-token-service/tests/execute.rs

```diff
@@ -16,7 +18,7 @@ use utils::{approve_gateway_messages, register_chains, setup_env, setup_its_toke
 use interchain_token::InterchainTokenClient;
 
 #[test]
-#[should_panic(expected = "Error(Contract, #1)")] // ExecutableError::NotApproved
+#[should_panic(expected = "Error(Contract, #22)")] // ContractError::NotApproved
```

Comment:
We can use the `assert_contract_err` macro now instead of testing for a panic

Reply:
I can address it in a follow up